kinetic/distribution.yaml

```diff
@@ -10489,6 +10489,21 @@ repositories:
       url: https://github.com/ipab-slmc/pybind11_catkin.git
       version: master
     status: developed
+  pyquaternion:
+    doc:
+      type: git
+      url: https://github.com/Achllle/pyquaternion.git
```

Comment:
So how does this relate to the pip key we have called [python-pyquaternion-pip](https://github.com/ros/rosdistro/blob/40e3f9c7b50b536af576a4fef23033e3ccc5b3c3/rosdep/python.yaml#L3371)?

Reply:
Same package.
My reasoning was based on @tfoote 's [comment on #23261](https://github.com/ros/rosdistro/pull/23261#pullrequestreview-331634341) that pip was not the right way to release. I also don't like the default behavior of `rosdep` that it runs `pip install` as root, which doesn't play well with `virtualenv`. Finally, I am considering releasing a cpp version, along with the cpp version of `dual_quaternions`, although this is not a near-term thing.
If the pip package is preferred or if there only can be one, happy to withdraw the PR.

Reply:
If we're going to package it here, we should just pull the pip version so that we don't get conflicts. Can you update this PR to remove the pip rosdep entry?

Reply:
done. I do want to note for full transparency that along with the additions to turn the package into a ROS package, I made some [minor changes that are not yet in the original package](https://github.com/KieranWynn/pyquaternion/compare/master...Achllle:master).

Reply:
That should be fine. Just be careful doing that with too much. We've done that in the past and been bitten when the upstream didn't take the change and then we were stuck maintaining it in our fork. If it's mostly packaging changes those are more common, the API changes are the most at risk of being a maintenance burden.

Reply:
Reverted one change and [opened a PR into the original repo for another fix](https://github.com/KieranWynn/pyquaternion/pull/56)